client/src/main/scala/com.gu.contentapi.client/utils/CapiModelEnrichment.scala

```diff
@@ -1,44 +1,50 @@
 package com.gu.contentapi.client.utils
 
 import com.gu.contentapi.client.model.v1._
+import com.gu.contentapi.client.utils.format._
+
 import java.time.OffsetDateTime
 import java.time.format.DateTimeFormatter
 
 object CapiModelEnrichment {
 
-  implicit class RichCapiDateTime(val cdt: CapiDateTime) extends AnyVal {
-    def toOffsetDateTime: OffsetDateTime = OffsetDateTime.parse(cdt.iso8601)
-  }
+  type ContentFilter = Content => Boolean
 
-  implicit class RichOffsetDateTime(val dt: OffsetDateTime) extends AnyVal {
-    def toCapiDateTime: CapiDateTime = CapiDateTime.apply(dt.toInstant.toEpochMilli, DateTimeFormatter.ISO_OFFSET_DATE_TIME.format(dt))
-  }
+  def getFromPredicate[T](content: Content, predicates: List[(ContentFilter, T)]): Option[T] =
+    predicates.collectFirst { case (predicate, t) if predicate(content) => t }
 
-  implicit class RichContent(val content: Content) extends AnyVal {
+  def tagExistsWithId(tagId: String): ContentFilter = content => content.tags.exists(tag => tag.id == tagId)
 
-    def designType: DesignType = {
+  def displayHintExistsWithName(displayHintName: String): ContentFilter = content => content.fields.flatMap(_.displayHint).contains(displayHintName)
 
-      val defaultDesignType = Article
+  def isLiveBloggingNow: ContentFilter = content => content.fields.flatMap(_.liveBloggingNow).contains(true)
 
-      type ContentFilter = Content => Boolean
+  val isImmersive: ContentFilter = content => displayHintExistsWithName("immersive")(content)
 
-      val isImmersive: ContentFilter = c => c.fields.flatMap(_.displayHint).contains("immersive")
+  val isMedia: ContentFilter = content => tagExistsWithId("type/audio")(content) || tagExistsWithId("type/video")(content) || tagExistsWithId("type/gallery")(content)
```

Comment:
[AR also has](https://github.com/guardian/apps-rendering/blob/bd7214635ea43813673e2fb40db783e45069bd58/src/item.ts#L237) `type/picture` in this category, see https://github.com/guardian/apps-rendering/pull/1146.

Reply:
Thanks for this. I'll add it to a card for changes to make on a future version